doc/documentation.rst

```diff
@@ -79,4 +79,21 @@ directory to the inferred package:
    (documentation
     (mld_files :standard \ wip))
 
+.. _odoc-options:
+
+Passing options to Odoc
+=======================
+
+.. code-block:: lisp
+
+    (env
+     (<profile>
+      (odoc <optional-fields>)))
+
+See :ref:`dune-env` for more details on the ``(env ...)`` stanza.
+``<optional-fields>`` are:
+
+- ``(warn-error <bool>)`` turns warnings into errors, if set to ``true``.
```

Comment:
To clarify, can you document that the default is `false`?

Reply:
Done